src/style.css

```diff
@@ -1 +1,64 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.searchbar {
+  margin: 20px 0;
```

Comment:
In this task its okay. But remember that if you want to configure the external geometry or positioning of a block, use separate classes or mix-ins to separate external properties (positioning, margins) from internal properties (padding). This approach helps avoid conflicts between element styles.